### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by klikinstall** [avocode/avocode#1631]
* **Version**: 2.15.3-w1e4dd12

* **Web/Desktop**: Web

* **Platform**: windows 6.1
* **Browser**: firefox 55.0

* **Avocode UID**: 177491
---

```
t/r._handleLineMouseUp@https://cdn.app.avocode.com/dist/v-6c25f75/js/avocode-web.min.js:101:763564
r@https://cdn.app.avocode.com/dist/v-6c25f75/js/avocode-web.min.js:79:608630

```

**Comment by jakubzitny** [comment]
Hi @klikinstall, thanks for the report. It's a bug and we'll try to fix it as soon as possible.

**Comment by jakubzitny** [comment]
Hi @klikinstall, this has been fixed in latest release 2.16.0. Please update and let us know if you have any more problems 😉
